playbooks/roles/elasticsearch/tasks/main.yml

```diff
@@ -28,20 +28,57 @@
     - "{{ elasticsearch_log_dir }}"
     - "{{ elasticsearch_cfg_dir }}"
 
+- name: Install Elasticsearch certificates
```

Comment:
@shimulch this breaks existing instances that didn't use ssl in elasticsearch, and it fails to add docs for how to deploy elasticsearch on a new server. (how is elasticsearch_keystore content generated?).

I'm trying to set up a new elasticsearch instance in https://github.com/open-craft/ansible-secrets/pull/246 ; would you mind explaining what's new so I can set it up? :)

Reply:
@swalladge For this part @kaizoku actually helped me to set up certificates. AFAIK The [elasticsearch-certutil](https://www.elastic.co/guide/en/elasticsearch/reference/7.11/certutil.html) has been used to generate the certificate. Then we converted it into a base64 string and set it as the value of ``elasticsearch_keystore``. 

We should have documented the process with this PR.

Reply:
Thanks @shimulch, will you create a follow up PR or ticket for adding docs?

Reply:
@swalladge ,
I've created https://tasks.opencraft.com/browse/BB-3918 to documenting this. Thanks for spotting this.